image/templates/helm/stackrox-secured-cluster/templates/collector.yaml

```diff
@@ -116,16 +118,46 @@ spec:
           name: etc-ssl
         - mountPath: /etc/pki/ca-trust/
           name: etc-pki-volume
-        - mountPath: /host
-          name: host-root-ro
+        - mountPath: /host/etc
+          name: etc-ro
+          readOnly: true
+        - mountPath: /host/proc
+          name: proc-ro
+          readOnly: true
+        - mountPath: /host/opt
+          name: opt-ro
+          readOnly: true
+        - mountPath: /host/run
+          name: run-ro
+          readOnly: true
+        - mountPath: /host/srv
+          name: srv-ro
+          readOnly: true
+        - mountPath: /host/sys
+          name: sys-ro
+          readOnly: true
+        - mountPath: /host/usr
+          name: usr-ro
+          readOnly: true
```

Comment:
I am wondering why do we mount now `/usr` when we used to mount only `/usr/lib`. Is this intended?

Reply:
It is intended. The collector and compliance containers now share volumes. The mount directories for the collector container are not changed, just the volumes used to provide those mounts (previously, collector directly mounted `/usr/lib` via a volume on the host `/usr/lib`, now we mount via `subPath` on the host `/usr` volume mount that is shared with the compliance container